generate-sql.js

```diff
@@ -0,0 +1,23 @@
+const fs = require('fs');
+
+const glue = fs.readFileSync("./pkg/rb62_wasm.js", { encoding: "utf8" });
+const buffer = fs.readFileSync("./pkg/rb62_wasm_bg.wasm");
+
+const bytes = Array.from(new Uint8Array(buffer.buffer));
+
+fs.writeFileSync("base62.sql", `\
+CREATE TEMP FUNCTION \`b62_to_hex\`(b62 STRING) RETURNS STRING LANGUAGE js AS '''
+${glue}
+const bytes = new Uint8Array(${JSON.stringify(bytes)});
+
+return wasm_bindgen(bytes).then(() => {
+    return wasm_bindgen.run(b62);
+});
```

Comment:
How come this would work? How do I tell this would call the `get_hex` or `get_b62` function?

Reply:
I had mistakenly thought you were only trying to call one function. I've updated it to export all the functions again.